openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -156,39 +149,39 @@ def _register_course_home_messages(request, course_id, user_access, course_start
         ).format(
             initial_tag=HTML(
                 '<div tabindex="0" aria-label="{aria_label_choice}" class="goal-option dismissible" '
-                'data-choice="{goal_key}">'
+                'data-choice="unsure">'
             ).format(
-                goal_key=GOAL_KEY_CHOICES.unsure,
                 aria_label_choice=Text(_("Set goal to: {choice}")).format(
-                    choice=GOAL_KEY_CHOICES[GOAL_KEY_CHOICES.unsure]
+                    choice=course_goal_options['unsure'],
                 ),
             ),
             choice=Text(_('{choice}')).format(
-                choice=GOAL_KEY_CHOICES[GOAL_KEY_CHOICES.unsure],
+                choice=course_goal_options['unsure'],
```

Comment:
I'm curious about these switches from GOAL_KEY_CHOICES.unsure to the string `unsure`.  It's usually better to use a constant than a string, even to make it simpler to track code.